spec/ParseFile.spec.js

```diff
@@ -1436,4 +1436,25 @@ describe('Parse.File testing', () => {
       expect(b.url).toMatch(/^http:\/\/localhost:8378\/1\/files\/test\/.*file.html$/);
     });
   });
+
+  it('works with array with more elements', async () => {
```

Comment:
Instead of adding a new test, could you extend the existing test above that seems to cover the same use case? And keep the `expect`s.

Reply:
I must admit I don't understand what do you mean by keeping the expects. I want to check if the Content-Type used in the test is validated against provided fileExtensions and it pass the validation. That's why I check if the promise is resolved.

Reply:
I think I got it :) Pushed the fix.